lib/msal-browser/src/cache/BrowserStorage.ts

```diff
@@ -94,7 +89,7 @@ export class BrowserStorage implements ICacheStorage {
             if (key.startsWith(`${Constants.CACHE_PREFIX}`) || key.startsWith(PersistentCacheKeys.ADAL_ID_TOKEN)) {
                 return key;
             }
-            return addInstanceId ? `${Constants.CACHE_PREFIX}.${this.clientId}.${key}` : `${Constants.CACHE_PREFIX}.${key}`;
+            return `${Constants.CACHE_PREFIX}.${this.clientId}.${key}`;
```

Comment:
+1. This is why I like new libraries. No roll overs :-D. Having said that we expect no cache support for implicit_flow to auth_code migration? We can get away with this if `msal@1.2.0` is mandated as the min version for SSO.

Reply:
I think that we need to have the stipulation that upgrading from msal@1.2.0 is required, but I can revisit this post-alpha/post-beta if we have any issues.

Reply:
Great. Thanks.